pkg/tablebackup/tablebackup.go

```diff
@@ -41,11 +41,11 @@ const (
 )
 
 type tableBackupState struct {
-	DeltasSinceBackupCount   uint32    // trigger next backup based on the deltas
-	SegmentsSinceBackupCount uint32    // trigger next backup based on the segments
-	LastWrittenMessage       time.Time // trigger next backup based on time
-	LatestCommitLSN          string    // assign correct name to the next segment
-	LatestFlushLSN           string    // do not write anything before or equal this point
+	DeltasSinceBackupCount   uint32    `yaml:"DeltasSinceBackupCount"`   // trigger next backup based on the deltas
```

Comment:
without tags the keys in the yaml file become all lowercased

Reply:
I actually don't see any issues with keys being lowercase in the yaml.